mim/commands/download.py

```diff
@@ -98,7 +98,8 @@ def download(package: str,
             from mmcv import Config
         except ImportError:
             raise ImportError(
-                'Please install mmengine to use the download command!')
+                '''Please install mmengine to use the download command:
+            `mim install mmengine`.''')
```

Comment:
```suggestion
                'Please install mmengine to use the download command: '
                '`mim install mmengine`.')
```
I think we don't need to wrap a new line ~
![image](https://user-images.githubusercontent.com/32220263/194837509-336a4a7e-6693-4ffd-86af-b771d7cdc0fb.png)